include/chains/tuple.hpp

```diff
@@ -6,7 +6,7 @@
 #ifndef CHAIN_TUPLE_HPP
 #define CHAIN_TUPLE_HPP
 
-namespace chain::inline v0 {
```

Comment:
This change is okay - I wanted to get rid of the plural for the project. Other non-chain suggestions for library and namespace names?